traffic_portal/app/src/common/modules/form/deliveryServiceJob/form.deliveryServiceJob.tpl.html

```diff
@@ -31,7 +31,12 @@
         <div class="helptext">
             <aside class="warning">
                 <h6>Warning</h6>
-                <p>Delivery service origin(s) must support IMS in order to safely invalidate content</p>
+                <p>REFRESH (preferred default) will mark the matching asset(s) as STALE and an IMS request of the asset(s) from the origin will be performed. Origins that fully adhere 
+                    to the <a href="https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.25">If-Modified-Since</a> 
+                    and <a href="https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.26">If-None-Match</a> specification can safely invalidate content using REFRESH.</p>
+                <p>REFETCH will mark the matching asset(s) as a cache MISS and a new full request of the asset(s) from the origin will be performed. 
+                    This operation is expensive and it is HIGHLY RECOMMENDED that the specified regex is very specific when using REFETCH.</p>
```

Comment:
emphasis should use [the emphasis HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em), or to convey contextual urgency rather than literal verbal stress, use [the strong importance element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em). All caps should not be used for either - and not just because there are better ways (and if you still want them in all caps that can be done accessibly through styling with `text-transform: uppercase` as a CSS rule) but because it causes accessibility problems. Screen-readers will read words in all caps letter-by-letter, which is why that should be reserved for cases where it's appropriate, like abbreviations, acronyms, and initialisms (which is why "IMS" is fine here).

Reply:
Done. Please double check the changes to CSS (`_form.scss`) to ensure they are correct and maintanable.